/**:
  ros__parameters:
    survey:
      # Survey area definition (lat, lon in degrees)
      area:
        corner1: [47.397742, 8.545594]  # Top-left
        corner2: [47.398242, 8.545594]  # Top-right
        corner3: [47.398242, 8.546594]  # Bottom-right
        corner4: [47.397742, 8.546594]  # Bottom-left
      
      # Survey parameters
      altitude: 50.0  # meters AGL
      speed: 5.0      # m/s
      line_spacing: 20.0  # meters between survey lines
      overshoot: 10.0     # meters overshoot at line ends
      
      # Camera/gimbal settings
      camera:
        trigger_distance: 5.0  # meters between photo triggers
        gimbal_pitch: -90.0    # degrees (straight down)
        
      # Safety parameters
      safety:
        rtl_altitude: 60.0     # Return to launch altitude
        geofence_radius: 200.0 # meters from home
        min_battery: 20.0      # percent